R/navbar-menu.R

```diff
@@ -2,11 +2,17 @@
 
 # Helpers for use within pkgdown itself - these must stay the same as the
 # yaml structure defined in vignette("customise")
-menu_submenu <- function(text, menu) {
+menu_submenu <- function(text, menu, icon = NULL, label = NULL, id = NULL) {
```

Comment:
I changed these files so that it's possible to generate the lightswitch menu using our internal code. The primary advantage is simplicity, but it also ensures that if the lightswitch is placed on the right, the menu is automatically right aligned.